src/codegen/riscv64/macro-assembler-riscv64.cc

```diff
@@ -3630,7 +3630,7 @@ void TurboAssembler::DsubOverflow(Register dst, Register left,
 }
 
 void TurboAssembler::MulOverflow(Register dst, Register left,
-                                 const Operand& right, Register overflow) {
+                                 const Operand &right, Register overflow) {
```

Comment:
`clang-format` should leave the `&` as-is (`Operand&`).

Reply:
I have fix it.